### PR TITLE
VITS HiFiTTS doc

### PR DESCRIPTION
**Pull request by treacker**
# What does this PR do ?

Update docs and model for HiFiTTS version

**Collection**: [TTS]

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [x] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: tutorials/tts/Vits_Training.ipynb [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "wqPMTEXXYUP4"
+   },
+   "source": [
+    "# License\n",
+    "\n",
+    "> Copyright 2020 NVIDIA. All Rights Reserved.\n",
```
Comment (XuesongYang) [inline tutorials/tts/Vits_Training.ipynb:25]: pls update the copyright header with `2023, NVIDIA CORPORATION & AFFILIATES`

File: tutorials/tts/Vits_Training.ipynb [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+    "# We will also need a few extra files for handling text.\n",
+    "!(mkdir -p scripts/tts_dataset_files \\\n",
+    "  && cd scripts/tts_dataset_files \\\n",
+    "  && wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/tts_dataset_files/cmudict-0.7b_nv22.10 \\\n",
+    "  && wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/tts_dataset_files/heteronyms-052722 \\\n",
+    "  && cd ..)\n",
+    "        \n",
+    "!(mkdir -p nemo_text_processing/text_normalization/en/data/whitelist/ \\\n",
+    "  && cd nemo_text_processing/text_normalization/en/data/whitelist/ \\\n",
+    "  && wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv \\\n",
```
Comment (XuesongYang) [inline tutorials/tts/Vits_Training.ipynb:173]: please confirm if you need this `whitelist` file explicitly. It has been applied by default after decoupling text normalization into a new repo from NeMo.

**Review by XuesongYang — CHANGES_REQUESTED** [review]
Thanks for your updates! Besides of comments above, could you please also add sections showing how to evaluate your VITS model, just like our FastPitch tutorial did?

File: tutorials/tts/Vits_Training.ipynb [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+    "name: &name VITS\n",
+    "\n",
+    "train_dataset: ???\n",
+    "validation_datasets: ???\n",
+    "sup_data_path: null\n",
+    "sup_data_types: null\n",
+    "\n",
+    "phoneme_dict_path: \"scripts/tts_dataset_files/cmudict-0.7b_nv22.10\"\n",
+    "heteronyms_path: \"scripts/tts_dataset_files/heteronyms-052722\"\n",
+    "whitelist_path: \"nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv\"\n",
```
Comment (XuesongYang) [inline tutorials/tts/Vits_Training.ipynb:230]: do we still need this line?

File: tutorials/tts/Vits_Training.ipynb [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+    "\n",
+    "```yaml\n",
+    "trainer:\n",
+    "  num_nodes: 1\n",
+    "  devices: 2\n",
+    "  accelerator: gpu\n",
+    "  strategy: ddp\n",
+    "  precision: 32\n",
+    "  # amp_backend: 'apex'\n",
+    "  # amp_level: 'O2'\n",
```
Comment (XuesongYang) [inline tutorials/tts/Vits_Training.ipynb:248]: do you want to make a further explanation on alternative options for `amp_backend` and `amp_level`? If not, please remove these two comments.

File: tutorials/tts/Vits_Training.ipynb [hunk 5]
```diff
@@ -0,0 +1,430 @@
...
+    "\n",
+    "```yaml\n",
+    "name: &name VITS\n",
+    "\n",
+    "train_dataset: ???\n",
+    "validation_datasets: ???\n",
+    "sup_data_path: null\n",
+    "sup_data_types: null\n",
+    "\n",
+    "phoneme_dict_path: \"scripts/tts_dataset_files/cmudict-0.7b_nv22.10\"\n",
```
Comment (XuesongYang) [inline tutorials/tts/Vits_Training.ipynb:228]: please correct the dict version: https://github.com/NVIDIA/NeMo/blob/main/scripts/tts_dataset_files/ipa_cmudict-0.7b_nv23.01.txt

File: tutorials/tts/Vits_Training.ipynb [hunk 6]
```diff
@@ -0,0 +1,430 @@
...
+    "{\"audio_filepath\": \"/path/to/audio2.wav\", \"text\": \"the other transcription\", \"duration\": 2.1}\n",
+    "...\n",
+    "```\n",
+    "Please note that the duration is in seconds.\n",
+    "\n",
+    "\n",
+    "Then you are ready to run your training script:\n",
+    "```bash\n",
+    "python vits.py train_dataset=YOUR_TRAIN.json validation_datasets=YOUR_VAL.json trainer.devices=-1\n",
+    "```"
```
Comment (XuesongYang) [inline tutorials/tts/Vits_Training.ipynb:333]: this section is a duplicate of Line 293.

File: tutorials/tts/Vits_Training.ipynb [hunk 7]
```diff
@@ -0,0 +1,430 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluating VITS\n",
+    "\n",
+    "Let's evaluate the quality of the VITS model.\n",
+    "\n",
+    "VITS is end-2-end model, so we don't need any additional models to generate audios."
```
Comment (XuesongYang) [inline tutorials/tts/Vits_Training.ipynb:344]: typo: `end-2-end` -> `end-to-end`.

File: tutorials/tts/Vits_Training.ipynb [hunk 8]
```diff
@@ -0,0 +1,430 @@
...
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.16"
```
Comment (XuesongYang) [inline tutorials/tts/Vits_Training.ipynb:425]: we try to avoid changes like these due to the differences between individual IDE metadata. Could you please clean up your jupyter notebook before updating the PR?

File: tutorials/tts/Vits_Training.ipynb [hunk 9]
```diff
@@ -0,0 +1,430 @@
...
+   "outputs": [],
+   "source": [
+    "test = \"Hey, this produces speech!\"\n",
+    "\n",
+    "model = VitsModel.from_pretrained(\"tts_en_lj_vits\")\n",
+    "\n",
+    "audio = evaluate_vits_ckpt(model, test)\n",
+    "\n",
+    "print(\"predicted audio\")\n",
+    "ipd.display(ipd.Audio(audio, rate=22050))"
```
Comment (XuesongYang) [inline tutorials/tts/Vits_Training.ipynb:396]: Adding a section for model evaluation is excellent, but only displaying synthesized audio signals is not enough to understand how far the difference from original audio. Could you please also help adding a mel-spectrogram comparison between synthesis and original?

**Review by XuesongYang — CHANGES_REQUESTED** [review]
thanks for updates. One question, are you using ARPABET or IPA units for VITS? If use ARPABET, the phoneme dict should be `cmudict-0.7b_nv22.10`, otherwise, `ipa_cmudict-0.7b_nv23.01.txt`

**Comment by treacker** [comment]
> 

I am using IPA

**Comment by XuesongYang** [comment]
> > 
> 
> I am using IPA

Gotcha! then please fix the version of ipa dict with `ipa_cmudict-0.7b_nv23.01.txt`. i still see you used `ipa_cmudict-0.7b_nv22.10.txt`

**Review by XuesongYang — APPROVED** [review]
